W09H01/src/test/pgdp/pools/TuplePoolTest.java

```diff
@@ -13,22 +13,25 @@ void insert() {
     String string0 = "Ladner";
     Integer int1 = 321;
     String string1 = "Jonas";
-    
-    Tuple<Integer, String> tuple0 = new Tuple<>(int0, string0);
-    Tuple<Integer, String> tuple0Duplicate = new Tuple<>(int0, string0);
-    Tuple<Integer, String> tuple1 = new Tuple<>(int1, string1);
-    Tuple<Integer, String> tuple2 = new Tuple<>(int1, string0);
-    
-    TuplePool<Integer, String> pool0 = new TuplePool<>();
-    
-    assertEquals(tuple0, pool0.insert(tuple0));
-    assertEquals(tuple1, pool0.insert(tuple1));
-    
-    assertEquals(tuple0, pool0.insert(tuple0Duplicate));
-    
-    assertEquals(tuple2, pool0.insert(tuple2));
+
+    Tuple<Object, Object> tuple0 = new Tuple<>(int0, string0);
```

Comment:
Why do you exactly change the type to Object? Is there any benefit?

Reply:
See https://github.com/LadnerJonas/PGdP-Tests-WS21-22/pull/39/files/739a1b7efdd2be463ed934b4c92cabbd6968b110#r776915949